avocado/utils/filelock.py

```diff
@@ -35,7 +35,7 @@ class FileLock(object):
 
     """
     Creates an exclusive advisory lock for a file.
-    All processes should would use and honour the advisory
+    All processes should would use and honor the advisory
```

Comment:
You're changing the spelling, but there's actually a grammar mistake in this sentence ("should would") which you're missing.

Honour and honor are both valid. The former is more common in en_GB, while the latter is en_US.